src/qz/ws/HttpAboutServlet.java

```diff
@@ -51,6 +78,66 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response) t
         }
     }
 
+    private void generateJsonResponse(HttpServletRequest request, HttpServletResponse response) {
+        JSONObject aboutData = AboutInfo.gatherAbout(request.getServerName());
+
+        try {
+            String certData = loadCertificate();
+            if (certData != null) {
+                aboutData.put("certificate", certData);
+            }
+        }
+        catch(Exception e) {
+            log.warn("Failed to load certificate data: {}", e.getMessage());
+        }
+
+        try {
+            response.setStatus(HttpServletResponse.SC_OK);
+            response.setContentType("application/json");
+            response.getOutputStream().write(aboutData.toString(JSON_INDENT).getBytes(StandardCharsets.UTF_8));
+        }
+        catch(Exception e) {
+            response.setStatus(HttpServletResponse.SC_INTERNAL_SERVER_ERROR);
+            log.warn("Exception occurred writing JSONObject {}", aboutData);
+        }
+    }
+
+    private void generateCertResponse(HttpServletRequest request, HttpServletResponse response) {
+        try {
+            String certData = loadCertificate();
+
+            if (certData != null) {
+                response.setStatus(HttpServletResponse.SC_OK);
+                response.setContentType("application/x-x509-ca-cert");
+
+                response.getOutputStream().print(certData);
+            } else {
+                response.setStatus(HttpServletResponse.SC_NOT_FOUND);
+            }
+        }
+        catch(Exception e) {
+            response.setStatus(HttpServletResponse.SC_INTERNAL_SERVER_ERROR);
+            log.warn("Exception occurred loading certificate: {}", e.getMessage());
+        }
+    }
+
+    private String loadCertificate() throws GeneralSecurityException, IOException {
+        Properties sslProps = DeployUtilities.loadTrayProperties();
+
+        if (sslProps != null) {
+            KeyStore jks = KeyStore.getInstance("jks");
+            jks.load(new FileInputStream(new File(sslProps.getProperty("wss.keystore"))), sslProps.getProperty("wss.storepass").toCharArray());
+
+            Certificate cert = jks.getCertificate("root-ca");
+
+            return "-----BEGIN CERTIFICATE-----" + System.lineSeparator() +
+                    new String(Base64.encodeBase64(cert.getEncoded(), true), StandardCharsets.UTF_8) +
```

Comment:
`System.lineSeparator()` can potentially conflict with `org.apache.commons.ssl.Base64` "isChunked" flag because Apache appears to default to `\r\n` on all platforms where as `System.lineSeparator()` defaults to `\n` on Unix, `\r\n` on Windows.